pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -489,6 +489,30 @@ func getConsoleTemplate(cr *api.KieApp) api.ConsoleTemplate {
 				template.DataGridAuth = *cr.Status.Applied.Objects.Console.DataGridAuth
 			}
 		}
+
+		// Enable EdgeTermination
+		if cr.Status.Applied.Objects.Console.TerminationRoute != nil && len(cr.Status.Applied.Objects.Console.TerminationRoute.Protocol) > 0 {
```

Comment:
isn't better apply the termination with the deepCopy method? this way you could save a few lines of code and some if conditions.

something similar to https://github.com/kiegroup/kie-cloud-operator/blob/main/pkg/controller/kieapp/defaults/auth.go#L88

Reply:
ok